docs/tutorial/kymotracking.rst

```diff
@@ -481,187 +481,136 @@ Since this algorithm is specifically looking for curvilinear structures, it can
-    DiffusionEstimate(value=3.8728160788405055, std_err=1.5207837420729884, num_lags=3, num_points=80, method='ols', unit='um^2 / s'),
-    DiffusionEstimate(value=4.9236019911012745, std_err=1.7399505893645122, num_lags=3, num_points=80, method='ols', unit='um^2 / s')]
-
-We can also directly determine them for an entire group by just invoking::
-
-    >>> tracks.estimate_diffusion(method="ols")
-    [DiffusionEstimate(value=7.804440367653842, std_err=2.527045387449447, num_lags=2, num_points=80, method='ols', unit='um^2 / s'),
-    DiffusionEstimate(value=3.8728160788405055, std_err=1.5207837420729884, num_lags=3, num_points=80, method='ols', unit='um^2 / s'),
-    DiffusionEstimate(value=4.9236019911012745, std_err=1.7399505893645122, num_lags=3, num_points=80, method='ols', unit='um^2 / s')]
-
-We can see that there is considerable variation in the estimates, which is unfortunately typical for diffusion coefficient estimates.
-By default, :func:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.estimate_diffusion` will use the optimal number of lags as specified in :cite:`michalet2012optimal`.
-You can however, override this optimal number of lags, by specifying a `max_lag` parameter::
-
-    >>> tracks.estimate_diffusion(method="ols", max_lag=30)
-    [DiffusionEstimate(value=11.949917925662831, std_err=10.394298104056345, num_lags=30, num_points=80, method='ols', unit='um^2 / s'),
-     DiffusionEstimate(value=4.904422868492953, std_err=4.3237670165379045, num_lags=30, num_points=80, method='ols', unit='um^2 / s'),
-     DiffusionEstimate(value=8.00626507619601, std_err=6.976860180814361, num_lags=30, num_points=80, method='ols', unit='um^2 / s')]
-
-Note however, that this will likely degrade your estimate (which you also see reflected in the estimated standard error).
-
-We can also plot the MSD estimates::
-
-    [track.plot_msd(marker='.') for track in tracks]
-
-.. image:: figures/kymotracking/msdplot_default_lags.png
-
-By default, this will use the optimal number of lags (which in this case seems to be around 3-4), but once again a `max_lag` parameter can be specified to plot a larger number of lags::
-
-    [track.plot_msd(max_lag=100, marker='.') for track in tracks]
-
-.. image:: figures/kymotracking/msdplot_100_lags.png
-
-It's not hard to see from this graph why taking too many lags results in unacceptably large variances (note how the traces diverge).
+This method is not recommended for very short tracks, as it will produce biased results in such scenarios.
+Care must be taken that while this estimate may be optimal for quantifying an individual track; it may lead to biased estimates when allowed to vary for each individual track.
+For more information on this, see the :doc:`theory section for diffusive processes</theory/diffusion/diffusion>`.
 
 .. note::
 
+    This method is not recommended for very short tracks, as it will produce biased results in such scenarios.
+
```

Comment:
How about merging the previous bit and this one into:

```
.. warning::

    It is not recommended to apply Ordinary Least Squares to invididual tracks if they are very short since it will produce biased results in such scenarios.
    Care must be taken that while the estimated number of lags may be optimal for quantifying an individual track; it may lead to biased estimates when averaging the results of several tracks with a different number of lags used for each individual track.
    For more information on this, see the :doc:`theory section for diffusive processes</theory/diffusion/diffusion>`.

    The uncertainty estimate and optimal number of lags obtained for Ordinary Least Squares relies on having a successful positional localization for every frame in a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack`.

    If frames are missing, then this method will issue a warning with the suggestion to refine tracks prior to estimation using :ref:`localization_refinement`.
    If this is not possible, please switch to :ref:`diffusion_cve`.
```

Reply:
👍